tests/Doctrine/Tests/Common/Persistence/ObjectManagerDecoratorTest.php

```diff
@@ -61,7 +61,7 @@ public function getMethodParameters()
      */
     public function testAllMethodCallsAreDelegatedToTheWrappedInstance($method, array $parameters, $isVoidMethod)
     {
-        $returnedValue = $isVoidMethod ? 'INNER VALUE FROM ' . $method : null;
+        $returnedValue = !$isVoidMethod ? 'INNER VALUE FROM ' . $method : null;
```

Comment:
Please just invert the ternary - adding a boolean negation just makes it more confusing